agrest/src/main/java/io/agrest/CompoundObjectId.java

```diff
@@ -63,6 +63,45 @@ public String toString() {
 		return mapToString(id);
 	}
 
+	@Override
+	public boolean equals(Object object) {
+		if (this == object) {
+			return true;
+		}
+
+		if (!(object instanceof CompoundObjectId)) {
+			return false;
+		}
+
+		CompoundObjectId compoundObjectId = (CompoundObjectId)object;
+
```

Comment:
Why is there so much code there? Everything below this line can be replaced with `id.equals(other.id)`... Also `equals` and `hashCode` require unit tests